src/artwork.c

```diff
@@ -1538,12 +1612,14 @@ source_item_stream_get(struct artwork_ctx *ctx)
   if ((len < 14) || (len > PATH_MAX)) // Can't be shorter than http://a/1.jpg
     goto out_url;
 
+/* 
+  //Remove check here and moved to artwork_read_byurl()
```

Comment:
Checking content-type is not the same as checking the URL, because the latter can save a request. But I see that with json we cannot count on file endings, so we have to make the request.